packages/cli/src/lib/snapshot/reportViewer.spec.ts

```diff
@@ -160,25 +159,25 @@ describe('ReportViewer', () => {
     });
 
     test
-      .skip() // TODO: remove
       .stdout()
       .do(() => {
         viewer.display();
       })
       .it('should print resource changes', (ctx) => {
-        const trimedStdout = ctx.stdout.replace(/\n\n/g, '');
+        // Remove padding added by cli-ux so we can test the text and not the padding on the line
+        const trimedStdout = ctx.stdout
+          .split('\n')
+          .map((s) => s.trimEnd())
+          .join('\n');
```

Comment:
You can save one operation and be 'OS safe' no matter fricking what with `$` instead of `\n`
![image](https://user-images.githubusercontent.com/12366410/121554471-342aa780-c9e0-11eb-9a1e-dbf054a3de94.png)
Because the text still has the \n at the beginning, you can skip the join afterward ;)

```suggestion
        const trimedStdout = ctx.stdout
          .split(/$/m)
          .map((s) => s.trimEnd());
```